app/models/ability.rb

```diff
@@ -0,0 +1,22 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    alias_action :index, :show, :to => :read
+
+    can :read, :all # permissions for every user, even if not logged in
+    if user.present?
+      can :create, [Comment, Sugest, User]
+      can [:update, :show, :edit], User, user_id: user.id
+      can [:read, :update, :show], Order, user_id: user.id
```

Comment:
`:read` sẽ là action nào gái

Reply:
alias_action :index, :show, :to => :read
read là action dùng chung cho index và show